### PR TITLE
fix: compatibility with old node versions

### PR DESCRIPTION
**Pull request by hyj1991**

### CODE REVIEW

File: src/profiler_data.cc [hunk 1]
```diff
@@ -5,8 +5,7 @@ namespace nodex {
   using v8::Context;
   using v8::Isolate;
 
-  ProfilerData::ProfilerData(v8::Local<Context> newContext, Isolate* isolate) {
-    context = newContext;
```
Comment (Widdershin) [inline src/profiler_data.cc:9]: Thanks for cleaning this up, at one point I was relying on `data->context` when I was using the v8 APIs instead of Nan.

Comment (hyj1991) [reply src/profiler_data.cc:9]: I have invited you to join this project :)

**Review by Widdershin — APPROVED** [review]
Looks good to me, and builds and tests fine on a few older versions on my machine.

Thanks for sorting out the version issues. I'll add a test for the multi-threading support when I get back to work next week.

**Comment by hyj1991** [comment]
> Looks good to me, and builds and tests fine on a few older versions on my machine.
> 
> Thanks for sorting out the version issues. I'll add a test for the multi-threading support when I get back to work next week.

OK

**Pull request merged**